platform/posix/transport/src/sockets_posix.c

```diff
@@ -98,6 +98,23 @@ static SocketStatus_t attemptConnection( struct addrinfo * pListHead,
                                          int * pTcpSocket,
                                          int32_t maxAttempts );
 
+/**
+ * @brief Connect to server using the provided address record.
+ *
+ * @param[in] pAddrInfo Address record of the server.
```

Comment:
`pAddrInfo` is also an output parameter, right?
```suggestion
 * @param[in, out] pAddrInfo Address record of the server.
```

Reply:
No. It is an input parameter